core/arch/arm/include/mm/core_mmu.h

```diff
@@ -375,23 +361,36 @@ enum teecore_tlb_op {
 int core_tlb_maintenance(int op, unsigned int a);
 
 /* Cache maintenance operation type */
-typedef enum {
-	DCACHE_CLEAN = 0x1,
-	DCACHE_AREA_CLEAN = 0x2,
-	DCACHE_INVALIDATE = 0x3,
-	DCACHE_AREA_INVALIDATE = 0x4,
-	ICACHE_INVALIDATE = 0x5,
-	ICACHE_AREA_INVALIDATE = 0x6,
-	WRITE_BUFFER_DRAIN = 0x7,
-	DCACHE_CLEAN_INV = 0x8,
-	DCACHE_AREA_CLEAN_INV = 0x9,
-	L2CACHE_INVALIDATE = 0xA,
-	L2CACHE_AREA_INVALIDATE = 0xB,
-	L2CACHE_CLEAN = 0xC,
-	L2CACHE_AREA_CLEAN = 0xD,
-	L2CACHE_CLEAN_INV = 0xE,
-	L2CACHE_AREA_CLEAN_INV = 0xF
-} t_cache_operation_id;
+enum cache_op {
+	DCACHE_CLEAN,
+	DCACHE_AREA_CLEAN,
+	DCACHE_INVALIDATE,
+	DCACHE_AREA_INVALIDATE,
+	ICACHE_INVALIDATE,
+	ICACHE_AREA_INVALIDATE,
+	DCACHE_CLEAN_INV,
+	DCACHE_AREA_CLEAN_INV,
+	L2CACHE_INVALIDATE,
```

Comment:
No, we don't. That's only legacy. Ok to clean it and move to `DCACHE_xxx` for both inner and outer cache data maintenance operations.